### PR TITLE
Fixed VCDM transformation algo, fixes #4

### LINKED ISSUE #4: Define transformation logic back to the base data model in W3C VCDM 2.0.
- omitting `type` claim in Presentation and treating it as a default value `VerifiablePresentations`

### PR DESCRIPTION
**Pull request by awoie**
Fixed VCDM transformation algo, fixes #4

### CODE REVIEW

File: draft-terbu-sd-jwt-vc.md [hunk 1]
```diff
@@ -524,68 +524,108 @@ model defined in [@VC-DATA], this specification defines a mapping algorithm for
...
 The following is a uni-directional transformation algorithm that takes in a
 VC-SD-JWT conformant to this specification and maps it
-to the corresponding properties in the W3C VCDM 2.0 [VCDM2.0]
+to the corresponding properties in the W3C VCDM 2.0 [@VC-DATA]
 which is based on a JSON-LD vocabulary. It includes specific handling for JWT
 claims used in this specification. The function returns a Verifiable
 Credential object in the W3C VCDM 2.0 format.
 
+Procedure:
+1. Let `payload` be the unsecured payload of the VC-SD-JWT reconstructed from the SD-JWT and Disclosures.
```
Comment (Sakurann) [inline draft-terbu-sd-jwt-vc.md:542]:
```suggestion
Procedure:
1. Validate SD-JWT following the steps defined in Section 6.2 of [@oauth-selective-disclosure-jwt].
1. Let payload be the payload of the VC-SD-JWT obtained in the previous step.
```

I thought we are calling it SD-JWT-VC and not VC-SD-JWT..?
I would prefer we move away from a VC WG specific terminology like secure/unsecured - signed, unsigned is simpler, no?
There is no `payload` claim in VCDM, right..?

File: draft-terbu-sd-jwt-vc.md [hunk 2]
```diff
@@ -524,68 +524,108 @@ model defined in [@VC-DATA], this specification defines a mapping algorithm for
...
+1. Let `VC` be an empty JSON object that represents the transformed Verifiable Credential:
+  - Set the `@context` property of `VC` to `"https://www.w3.org/ns/credentials/v2"`.
```
Comment (Sakurann) [inline draft-terbu-sd-jwt-vc.md:544]:
```suggestion
1. Let `vc` be an empty JSON object that represents the transformed Verifiable Credential:
  - Set the `@context` property of `vc` to `"https://www.w3.org/ns/credentials/v2"`.
```

why uppercase..?

Comment (awoie) [reply draft-terbu-sd-jwt-vc.md]: @Sakurann `VC` is a variable that contains a JSON object in that case which is also the resulting W3C VC. I made it lower-case now. Does that work for you?

File: draft-terbu-sd-jwt-vc.md (hunk 1 above)
Comment (awoie) [reply draft-terbu-sd-jwt-vc.md:542]: Can we please have the naming discussion in a separate issue? This PR does not introduce a new name, only the algorithm.

Comment (danielfett) [reply draft-terbu-sd-jwt-vc.md:542]: Agree with @Sakurann's proposals but to move naming discussion elsewhere.

Comment (awoie) [reply draft-terbu-sd-jwt-vc.md:542]: I created this issue https://github.com/vcstuff/draft-terbu-sd-jwt-vc/issues/70, resolving this discussion.

File: draft-terbu-sd-jwt-vc.md (hunk 2 above)
Comment (awoie) [reply draft-terbu-sd-jwt-vc.md]: We will use italic instead of double quotes for variable names.

**Review by danielfett — APPROVED** [review]

**Pull request merged**